README.md

```diff
@@ -52,3 +52,7 @@ also make use of sequential I/O operations only.
 
 The `mtbl` file format was changed in version 1.0.0. Older versions cannot read
 the new file format, but newer versions can read both formats.
+
```

Comment:
I think we can leave this out of README, it's mostly ChangeLog material, and references external software.

The note on the 1.0.0 change was warranted IMO, as that was a compatibility breaking change.

Reply:
Pushed